### PR TITLE
Add rice

### PR DESCRIPTION
**Pull request by ornobalam**

### CODE REVIEW

**Comment by nspope** [comment]
Thanks @ornobalam! There's a conflict because you made this commit is on top of your old feature branch, which already got squashed and merged. So, could you please:
```bash
# 1. navigate to your local copy of stdpopsim and pull so that the "main" branch is up-to-date
git checkout main
git pull https://github.com/popsim-consortium/stdpopsim main
git log | head
```
You should see "commit 74b8e84...." as the top entry of the log (it's the squashed, merged work from your last PR). Then:
```bash
# 2. make a new branch with your updates (in case something goes wrong, you won't lose work)
git fetch https://github.com/popsim-consortium/stdpopsim pull/1462/head:add_rice-backup

# 3. switch to your "add_rice" branch
git checkout add_rice
git log | head
```
Now, you should see `commit e0f50a02...` (your mutation rate fixes) as the top entry of the log. Next:
```bash
# 4. rebase your last commit onto the up-to-date main branch.
# in other words, take the work you just did (in your old, obsolete branch), and apply it to the main branch
git rebase --onto main add_rice~1 add_rice
git log | head
```
You should see your "fixed mutation rate and pop size" commit, followed by "74b8e..." (which is the latest commit on main). Make sure this is the case, because the next step is going to overwrite your remote copy of add_rice (although you'll still have the backup you just created). Finally:
```bash
# 5. force push to update this PR
git push -u origin add_rice --force
```

Hopefully that makes sense! Let me know if you have any questions. And sorry for the hassle-- sometimes the simplest things become complicated with git.

**Comment by nspope** [comment]
Once you rebase, there's also [this comment](https://github.com/popsim-consortium/stdpopsim/issues/1458#issuecomment-1512182504) from @minesrebollo

**Comment by ornobalam** [comment]
@nspope Thank you for laying it out so clearly! I think the rebasing worked, and I made the changes suggested by @minesrebollo.
